include/kernels/field_strength_tensor.cuh

```diff
@@ -169,14 +172,18 @@ namespace quda
       F *= static_cast<typename Arg::Float>(0.125); // 18 real multiplications
       // 36 floating point operations here
     }
-    
+
     int munu_idx = (mu * (mu - 1)) / 2 + nu; // lower-triangular indexing
     arg.f(munu_idx, idx, parity) = F;
   }
 
-  template <typename Arg> struct ComputeFmunu {
+  template <typename Arg_> struct ComputeFmunu : computeFmunuCoreOps {
```

Comment:
Similar comment as with clover derivatives, I think we should just have this inherit from `KernelOps<thread_array<int, 4>>`